resilience4j-spring-boot2/src/main/java/io/github/resilience4j/circuitbreaker/monitoring/endpoint/CircuitBreakerServerSideEvent.java

```diff
@@ -0,0 +1,115 @@
+        Seq<Flux<CircuitBreakerEvent>> eventStreams = circuitBreakerRegistry.getAllCircuitBreakers()
+            .map(
+                circuitBreaker -> toFlux(circuitBreaker.getEventPublisher())
+            );
+        Function<CircuitBreakerEvent, String> data = getCircuitBreakerEventStringFunction();
+        return Flux.merge(eventStreams).map(
+            cbEvent -> ServerSentEvent.<String>builder()
+                .id(cbEvent.getCircuitBreakerName())
+                .event(cbEvent.getEventType().name())
+                .data(data.apply(cbEvent))
+                .build()
+        );
+    }
+
+    @ReadOperation(produces = "text/event-stream")
+    public Flux<ServerSentEvent<String>> getEventsFilteredByCircuitBreakerName(
+        @Selector String name) {
+
+        CircuitBreaker circuitBreaker = getCircuitBreaker(name);
+        Flux<CircuitBreakerEvent> eventStream = toFlux(circuitBreaker.getEventPublisher());
+        Function<CircuitBreakerEvent, String> data = getCircuitBreakerEventStringFunction();
+        return eventStream.map(
+            cbEvent -> ServerSentEvent.<String>builder()
+                .id(cbEvent.getCircuitBreakerName())
+                .event(cbEvent.getEventType().name())
+                .data(data.apply(cbEvent))
+                .build()
+        );
+    }
+
+    @ReadOperation(produces = "text/event-stream")
+    public Flux<ServerSentEvent<String>> getEventsFilteredByCircuitBreakerNameAndEventType(
+        @Selector String name, @Selector String eventType) {
+
+        CircuitBreaker circuitBreaker = getCircuitBreaker(name);
+        Flux<CircuitBreakerEvent> eventStream = toFlux(circuitBreaker.getEventPublisher())
+            .filter(
+                event -> event.getEventType() == CircuitBreakerEvent.Type.valueOf(eventType.toUpperCase())
+            );
+        Function<CircuitBreakerEvent, String> data = getCircuitBreakerEventStringFunction();
+        return eventStream.map(cbEvent -> ServerSentEvent.<String>builder()
+            .id(cbEvent.getCircuitBreakerName())
+            .event(cbEvent.getEventType().name())
+            .data(data.apply(cbEvent))
+            .build()
+        );
+    }
+
+    private Function<CircuitBreakerEvent, String> getCircuitBreakerEventStringFunction() {
+        ObjectMapper jsonMapper = new ObjectMapper();
+        return cbEvent -> {
+            try {
+                return jsonMapper.writeValueAsString(
+                    CircuitBreakerEventDTOFactory.createCircuitBreakerEventDTO(cbEvent)
+                );
+            } catch (JsonProcessingException e) {
+                /* ignore silently */
+            }
+            return "";
+        };
+    }
+
+    private CircuitBreaker getCircuitBreaker(String circuitBreakerName) {
+        CircuitBreaker circuitBreaker = circuitBreakerRegistry.circuitBreaker(circuitBreakerName);
+        if (circuitBreaker == null && !circuitBreaker.getName().equalsIgnoreCase(circuitBreakerName)) {
```

Comment:
I think this doesn't need to be checked. 
`CircuitBreakerRegistry` create or get when you request a circuitbreaker. 
Therefore, it cannot be null, and it must be the same as the requested name.